components/landing/TheQuickStart/StartLocally.vue

```diff
@@ -28,42 +29,39 @@
             @click="selectOption(choicesGroup, option)"
           >
             {{ option }}
-          </cv-button>
-        </cv-button-set>
+          </bx-btn>
+        </div>
       </div>
       <div>
-        <cv-accordion
-          v-if="prerequisitesToInstallQiskit()"
+        <bx-accordion
+          v-if="prerequisitesToInstallQiskit"
           class="start-locally__prerequisites-section"
         >
-          <cv-accordion-item>
-            <template #title>
-              Installing from source requires that you have the Rust compiler on
-              your system
-            </template>
-            <template #content>
-              <lazy-prerequisites-for-windows
-                v-if="selectedOs === OPERATING_SYSTEMS.windows"
-              />
-              <lazy-prerequisites-for-linux-mac v-else />
-            </template>
-          </cv-accordion-item>
-        </cv-accordion>
+          <bx-accordion-item
+            title-text="Installing from source requires that you have the Rust compiler on your system"
+          >
+            <LazyLandingTheQuickStartPrerequisitesForWindows
```

Comment:
This component is being redefined at the moment, so it will be removed sooner than later

Reply:
Since we don't know yet how this will look in the future, is it OK to leave it like this for now?

Later, once we know what we want to have instead we could revisit this and make the necessary changes.

I was thinking about adding a "TODO" but since it hasn't yet been decided what will be done, I think there is no to do for now.

What do you think?

Reply:
We will have a new component. Another Quick Start but redesigned

Reply:
As pointed out, there will be a major redesign with https://github.com/Qiskit/qiskit.org/issues/2856, so it makes sense to remove the whole component and leave a TODO instead. I'll do that now.

Reply:
Changed in https://github.com/Qiskit/qiskit.org/pull/2986/commits/dd25e0d2517341fc3a36b6a2ac73ed90e6733a82